packages/plugin-hardhat/hardhat.config.js

```diff
@@ -31,4 +31,8 @@ module.exports = {
       'contracts/GapV2_Bad.sol': override,
     },
   },
+  defender: {
+    apiKey: 'KEY',
+    apiSecret: 'SECRET',
+  },
```

Comment:
These should not be required. Are they?

Reply:
This was needed for a test, but it's kind of pointless so I removed it in https://github.com/OpenZeppelin/openzeppelin-upgrades/pull/863/commits/d955c71fb871cd42006fa07c9756608e79a1d4d2 (and the rest of that test is a duplicate of what we already had for Platform).